File-Formats/PDF/Working-with-DigitalSignature.md

```diff
@@ -5700,7 +5032,7 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## Remove existing digital signatures from a PDF document 
 
-The following code example illustrates how to remove existing digital signatures from a PDF document. 
+The following code example illustrates how to remove existing digital signatures from a PDF document using [Remove](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Interactive.PdfFieldCollection.html#Syncfusion_Pdf_Interactive_PdfFieldCollection_Remove_Syncfusion_Pdf_Interactive_PdfField_) method. 
```

Comment:
The following code example illustrates how to remove existing digital signatures from a PDF document using [Remove](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Interactive.PdfFieldCollection.html#Syncfusion_Pdf_Interactive_PdfFieldCollection_Remove_Syncfusion_Pdf_Interactive_PdfField_) method in [PdfLoadedFormFieldCollection]() class.

Reply:
Changed the content